### PR TITLE
Ios send token patch

### PR DESCRIPTION
**Pull request by deepak786**
It may solve the issue https://github.com/v3rm0n/intercom_flutter/issues/149

### CODE REVIEW

File: intercom_flutter/ios/Classes/IntercomFlutterPlugin.m [hunk 1]
```diff
@@ -188,4 +190,40 @@ - (ICMUserAttributes *) getAttributes:(FlutterMethodCall *)call {
     
     return attributes;
 }
+
+- (NSData *) createDataWithHexString:(NSString*)inputString {
```
Comment (Zazo032) [inline intercom_flutter/ios/Classes/IntercomFlutterPlugin.m:194]: As I understand, this function takes a string and returns that string in hexadecimal format, right? Is there any language function/feature that does that?

**Review by Zazo032 — APPROVED** [review]
LGTM

File: intercom_flutter/ios/Classes/IntercomFlutterPlugin.m (hunk 1 above)
Comment (deepak786) [reply intercom_flutter/ios/Classes/IntercomFlutterPlugin.m:194]: I don't know. But As I checked the other PR that is also doing the same thing https://github.com/v3rm0n/intercom_flutter/pull/79/files#:~:text=byte_chars-,%5B,-0%5D%20%3D%20%5Bstring%20characterAtIndex%3Ai

**Comment by deepak786** [comment]
@Zazo032 could you please update the package intercom_flutter on pub.dev?

**Pull request merged**